### ISSUE TITLE
Modals are very hard to read on high contrast theme

### ISSUE THREAD

**Issue by tonyanziano** [Microsoft/BotFramework-Emulator#1312]
## Version
Latest source

## Describe the bug
The over-app modals are all very difficult to read on high contrast theme.

## To Reproduce
1. Set theme to High Contrast
2. Edit an endpoint, service, or create a new bot and look at the modal.

![image](https://user-images.githubusercontent.com/3452012/52661979-1acccb80-2eb8-11e9-8b2c-89979fd2630c.png)

## Expected behavior
The text should be easily readable.

[bug]

**Comment by tonyanziano** [comment]
Addressed in #1402
